client/src/extension.node.ts

```diff
@@ -20,8 +20,7 @@ export function activate(context: ExtensionContext) {
     // Register the server for all document types
     documentSelector: [{ language: "openfga" }, { language: "yaml-store-openfga" }],
     synchronize: {
-      // Notify the server about file changes to '.clientrc files contained in the workspace
-      fileEvents: workspace.createFileSystemWatcher("**/.clientrc"),
+      fileEvents: workspace.createFileSystemWatcher("**/*.{fga.yaml,fga,openfga,openfga.yaml,yaml,json,csv}"),
```

Comment:
Minor: This code seems duplicated in browser and in node, can we perhaps extract it to a single place?

This can be in a separate follow up PR

Reply:
Created an issue: https://github.com/openfga/vscode-ext/issues/154